### ISSUE TITLE
When fine-tuning the model, the loss is greater than 1

### ISSUE THREAD

**Issue by FelexTriz** [meta-llama/llama-recipes#296]
### System Info

pytorch2.0.1
cuda11.8
gpu 3090

### Information

- [ ] The official example scripts
- [x] My own modified scripts

### 🐛 Describe the bug

here's my hyperparameters
python -m llama_recipes.finetuning  
--use_peft
 --peft_method lora 
--quantization 
--use_fp16 
--model_name /root/SHARE/Models/Llama-2-7b-hf  
--output_/root/SHARE/Models/llama_finetuning_7B-chat 
--dataset alpaca_dataset 
--batch_size_training 30
loss was about 0.9 at first ,but it bigger than 1 at steps 2500

### Error logs

![image](https://github.com/facebookresearch/llama-recipes/assets/82858160/b0e64723-69f6-4c2f-82b9-b1ba56e7ed2d)

### Expected behavior

the loss it shoult be lower,right?

**Comment by CreamyLong** [comment]
I also use RTX 3090 but the training occupies 22G memory when the batch_size_training=1. How do you set --batch_size_training 30?

**Comment by init27** [comment]
Could you kindly try our updated Fine-tunning scripts and re-open if you still face the same issue?
